### ISSUE TITLE
0.35.1 beta 2 (Windows): Error creating DAG for FiroPow

### ISSUE THREAD

**Issue by UselessGuru** [andru-kun/wildrig-multi#179]
`wildrig.exe --algo firopow --api-port 4004 --url stratum+tcps://firopow.mine.zergpool.com:13001 --user *****************************--pass *********************,c=BTC,pl=0 --multiple-instance --opencl-platforms 1 --opencl-devices 0`

```
[08:57:26] initializing GPUs...
[08:57:26] GPU #1: AMD Radeon RX 5700 XT [busID: 9] [arch: gfx1010]
[08:57:26] threads: 1, intensity: 23, worksize: 0/256, cu: 40, mem: 8176Mb
[08:57:27] use pool firopow.mine.zergpool.com:13001 103.249.70.7
[08:57:27] new job from firopow.mine.zergpool.com:13001 diff 42.95M/29.16T
[08:57:27] block: 587,682       epoch: 452      job target: 0x00000063ff9c0000
[08:57:30] new job from firopow.mine.zergpool.com:13001 diff 42.95M/29.16T
[08:57:30] block: 587,683       epoch: 452      job target: 0x00000063ff9c0000
[08:57:31] kernel 1 compiled for gfx1010
[08:57:31] Error CL_INVALID_KERNEL when calling clSetKernelArg for kernel 'ethash_calculate_dag_item', argument 1
[08:57:32] Error UNKNOWN_ERROR when initialize DAG
[08:57:39] Error CL_INVALID_KERNEL when calling clSetKernelArg for kernel 'ethash_calculate_dag_item', argument 1
[08:57:40] Error UNKNOWN_ERROR when initialize DAG
[08:57:49] rejected(0/1)        diff 1.00       GPU#1   (101 ms)
[08:57:49] reason: Low difficulty share
```

**Comment by UselessGuru** [comment]
What is the min memory reqd. for nexapow?

**Comment by andru-kun** [comment]
> What is the min. memory required for nexapow?

3Gb is recommended to fit 1.5Gb precompute table and other buffers, 1Gb is minimum I guess ) But such old cards won't perform good.

**Comment by andru-kun** [comment]
should be fixed since 0.35.2 beta
